rotkehlchen/globaldb/handler.py

```diff
@@ -1203,6 +1227,144 @@ def get_historical_price_data(source: HistoricalPriceOracle) -> List[Dict[str, A
              'to_timestamp': entry[3],
              } for entry in query]
 
+    @staticmethod
+    def completly_rebuild_assets_list(user_db_conn: sqlite3.Connection) -> Tuple[bool, str]:
```

Comment:
```suggestion
    def completely_rebuild_assets_list(user_db_conn: sqlite3.Connection) -> Tuple[bool, str]:
```

I may also prefer a more unambiguous name. Like: `hard_reset_assets_list()`